tools/ci.py

```diff
@@ -21,6 +21,32 @@
 ci = command_group(name="ci", help="CI Related Commands", description=__doc__)
 
 
+@ci.command(
+    name="print-gh-event",
+)
+def print_gh_event(ctx: Context):
+    """
+    Pretty print the GH Actions event.
+    """
+    gh_event_path = os.environ.get("GITHUB_EVENT_PATH") or None
+    if gh_event_path is None:
+        ctx.warn("The 'GITHUB_EVENT_PATH' variable is not set.")
+        ctx.exit(1)
+
+    if TYPE_CHECKING:
+        assert gh_event_path is not None
+
+    try:
+        gh_event = json.loads(open(gh_event_path).read())
+    except Exception as exc:
+        ctx.error(f"Could not load the GH Event payload from {gh_event_path!r}:\n", exc)
+        ctx.exit(1)
+
+    ctx.info("GH Event Payload:")
+    ctx.print(gh_event, soft_wrap=True)
```

Comment:
Is there a possibility that this could print secure data?

Reply:
No, the github api events don't carry secure/private data